src/status_im/contexts/wallet/collectible/tabs/overview/view.cljs

```diff
@@ -0,0 +1,72 @@
+(ns status-im.contexts.wallet.collectible.tabs.overview.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme]
+    [react-native.core :as rn]
+    [status-im.contexts.wallet.collectible.tabs.overview.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- traits-section
+  []
+  (let [traits (rf/sub [:wallet/last-collectible-details-traits])]
+    (when (pos? (count traits))
+      [rn/view
+       [quo/section-label
+        {:section         (i18n/label :t/traits)
+         :container-style style/traits-title-container}]
+       [rn/flat-list
+        {:render-fn               (fn [{:keys [trait-type value]}]
+                                    [quo/data-item
+                                     {:description     :default
+                                      :card?           true
+                                      :status          :default
+                                      :size            :default
+                                      :title           trait-type
+                                      :subtitle        value
+                                      :container-style style/traits-item}])
+         :data                    traits
+         :key                     :collectibles-list
+         :key-fn                  :id
+         :num-columns             2
+         :content-container-style style/traits-container}]])))
+
+(defn- info
+  []
+  (let [chain-id        (rf/sub [:wallet/last-collectible-details-chain-id])
+        network         (rf/sub [:wallet/network-details-by-chain-id
+                                 chain-id])
+        network-keyword (get network :network-name)
```

Comment:
The naming here is a bit confusing. `network-keyword` was the hint for me that something was odd.

Here's a suggestion where:

- We destructure because we don't need the entire result from the sub.
- We `or` `network-name` coming off of the subscription before calling `name` on it because if it's nil an exception will be thrown.
- We name the transformed network name as `subtitle`, because that's the name of the prop down there that will use the capitalized text.

```diff
modified   src/status_im/contexts/wallet/collectible/tabs/overview/view.cljs
@@ -35,11 +35,9 @@
 
 (defn- info
   []
-  (let [chain-id        (rf/sub [:wallet/last-collectible-details-chain-id])
-        network         (rf/sub [:wallet/network-details-by-chain-id
-                                 chain-id])
-        network-keyword (get network :network-name)
-        network-name    (string/capitalize (name network-keyword))]
+  (let [chain-id               (rf/sub [:wallet/last-collectible-details-chain-id])
+        {:keys [network-name]} (rf/sub [:wallet/network-details-by-chain-id chain-id])
+        subtitle               (string/capitalize (name (or network-name "")))]
     [rn/view
      {:style style/info-container}
      [rn/view {:style style/account}
@@ -60,8 +58,8 @@
         :status        :default
         :size          :default
         :title         (i18n/label :t/network)
-        :network-image (quo.resources/get-network network-keyword)
-        :subtitle      network-name}]]]))
+        :network-image (quo.resources/get-network network-name)
+        :subtitle      subtitle}]]]))
```

Reply:
Done